src/styles/utils/_functions.scss

```
@@ -0,0 +1,15 @@
+@function calcLeft($arrowWidth, $clockWidth) {
```

function names should be more descriptive
and do you really need a function for these calculation?
```suggestion
@function calculateLeftIndent($arrowWidth, $clockWidth) {
```